cmd/nirvana/api/api.go

```diff
@@ -77,30 +77,27 @@ func (o *apiOptions) Install(flags *pflag.FlagSet) {
 }
 
 func (o *apiOptions) Validate(cmd *cobra.Command, args []string) error {
+	if len(args) < 1 {
+		return fmt.Errorf("must specify a project path")
```

Comment:
If you do not specify args, you should use the default path, I think you can add a line of log to tell the user.